src/fmod_studio.rs

```diff
@@ -66,4 +67,21 @@ impl FmodStudio {
 
         studio
     }
+
+    pub fn play_event_at(&self, transform: Transform, path_or_id: &str) {
+        let event_description = self.0.get_event(path_or_id).unwrap();
+        let event_instance = event_description.create_instance().unwrap();
```

Comment:
Creating a new instance ad-hoc every time this function is called smells like a performance issue to me.

> When an event instance is created, any required non-streaming sample data is loaded asynchronously.

This would incur loading and unloading the sample data at the mercy of FMOD. I can imagine this can possibly cause delays when trying to play the sound immediately, as a one-shot usually requires.

Reply:
Ah yeah you're right, the API guide even calls this out. 

In practice it doesn't seem to be a problem: https://discord.com/channels/691052431525675048/1215038273445961758/1215317674595192852

But I realize now that users can already create their own play_event_at function with the way things are, so I think we can leave this PR for now until we fix https://github.com/Salzian/bevy_fmod/issues/20

Reply:
I think we should leave things as they are for now to ensure samples are loaded when needed. We could add a utility function that combines playing a sound and moving the entity to the desired position.